Projects/App/Sources/UI/Review/ViewModel/ReviewRegisterViewModel.swift

```diff
@@ -54,61 +54,71 @@ extension ReviewRegisterViewModel: ErrorMapper {
     
     func uploadImage(with image: UIImage?) {
         let imageData = image?.pngData()
+        
```

Comment:
p1)
우리의 돈은 소중하다
압축 부탁드립니다! + 이렇게 하니 속도 문제가 개선이 되었습니다 🥳

일단 0.2로 가고 사진 손실이 많이나면 올립시다
(야간모드, 인물사진, raw 파일을 모두 고려한 내용입니다)
```suggestion
        let imageData = image?.jpegData(compressionQuality: 0.2)
```

Reply:
그리고 압축코드를 네트워크 레이어에 넣는게 좋을 것 같아요

Reply:
7570abb3fded86409195e0846e782bc65cc2af85
압축 코드 추가 및 코드 usecase로 이동 완료했습니다!